frontend/server/src/Controllers/Certificate.php

```diff
@@ -0,0 +1,29 @@
+<?php
+
+namespace OmegaUp\Controllers;
+
+/**
+ * CertificateController
+
+ * @psalm-type CertificateDetailsPayload=array{uuid: string}
+ */
+class Certificate extends \OmegaUp\Controllers\Controller {
+    /**
+     * @return array{smartyProperties: array{payload: CertificateDetailsPayload, title: \OmegaUp\TranslationString}, entrypoint: string}
+     *
+     * @omegaup-request-param null|string $uuid
+     */
+    public static function getDetailsForSmarty(\OmegaUp\Request $r) {
+        return [
+            'smartyProperties' => [
+                'payload' => [
+                    'uuid' => strval($r['uuid']),
```

Comment:
Acá era como te indicó Luis Héctor:
```suggestion
                    'uuid' => $r->ensureString('uuid'),
```
Con eso nos aseguramos que `uuid` es una cadena y no tenemos que forzar tipos. Además de que PSALM ya no se quejará.